src/Sarif.Multitool/PageOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("page", HelpText = "Extract a subset of results from a source SARIF file.")]
+    internal class PageOptions
```

Comment:
>PageOptions [](start = 19, length = 11)

are 'inline', 'force', 'pretty-print' useful for this command? they tend to be standard for other log file processing/producing things.

Reply:
Added force, with different default and explanation.
Pretty-Print can't be implemented here.
Inline seems like a bad idea for Page because the map would be built for an immediately invalidated file and all data outside the desired range would be lost.